app/jobs/subscription_confirm_job.rb

```diff
@@ -93,6 +94,14 @@ def send_confirmation_email(order)
     SubscriptionMailer.confirmation_email(order).deliver_now
   end
 
+  def send_payment_authorization_emails(order)
+    order.payments.each do |payment|
+      if payment.cvv_response_message.present?
+        PaymentMailer.authorize_payment(payment).deliver
```

Comment:
There's one or two more places where the deprecated `#delete` method is being called still. `#deliver_later?`

Reply:
980ac6ff2ddbc6d7767f80fb669dc8b77aa15aae